src/updateChecker.js

```
@@ -0,0 +1,112 @@
+      const latestMajor = semver.major(latestVersion);
+      if (options.autoInstall && currentMajor === latestMajor) {
+        await performUpdate({ tag: options.tag });
+        return true;
+      }
+
+      // If not auto-installing, prompt the user
+      const answer = await promptForUpdate();
+      if (answer) {
+        await performUpdate({ tag: options.tag });
+        return true;
+      }
```

Biome

[error] This code is unreachable



(lint/correctness/noUnreachable)